### PR TITLE
Bump k8s min version

### LINKED ISSUE #622: Min Kubernetes Version
**Describe the Task**
Please consider changing K8s version requirement, currently it [says](https://docs.pixielabs.ai/installing-pixie/requirements/#kubernetes) `v1.16+`. However when installing on 1.20 I'm getting error:

``` bash
time="2022-10-12T10:39:14Z" level=error msg="Failed to deploy Vizier deps" error="no matches for kind \"PodDisruptionBudget\" in version \"policy/v1\""
```

Please consider changing to `v1.21+` as per [docs](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125)

### PR DESCRIPTION
**Pull request by vihangm**
Fixes https://github.com/pixie-io/pixie/issues/622
Signed-off-by: Vihang Mehta <vihang@pixielabs.ai>

### CODE REVIEW

**Review by htroisi — APPROVED** [review]

**Comment by ghost** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/pixie-io/docs.px.dev/pull/231#issuecomment-1281190108" title="Approved">htroisi</a>*, *<a href="https://github.com/pixie-io/docs.px.dev/pull/231#" title="Author self-approved">vihangm</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=pixie-io%2Fdocs.px.dev).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/pixie-io/docs.px.dev/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
